library/ziti.c

```diff
@@ -1336,10 +1337,6 @@ static void api_session_cb(ziti_api_session *session, const ziti_error *err, voi
 
     if (session) {
 
-        update_ctrl_status(ztx, ZITI_PARTIALLY_AUTHENTICATED, NULL);
-
-        ZTX_LOG(DEBUG, "%s successfully => api_session[%s]", ztx->api_session ? "refreshed" : "logged in", session->id);
-
         ziti_set_api_session(ztx, session);
```

Comment:
`ziti_set_api_session` was always called and has the appropriate test for auth queries inside it. just moved `update_ctrl_status` into there. I'll put that debug message back though. didn't really want to delete that